src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -260,20 +266,37 @@ class adminLogs extends viewModelBase {
         eventsCollector.default.reportEvent("admin-logs", "connect");
         const ws = new adminLogsWebSocketClient(this.configuration(), data => this.onData(data));
         this.liveClient(ws);
-        
-        this.headerSeen = false;
+
+        this.liveClient().connectionOpened.subscribe((opened) => {
```

Comment:
please keep in mind that here you subscribe to live client which is under this observable. When we disconnect we null this.liveClient() so such subscription needs to be created again once we reconnect, right?

Reply:
This is the place that you get to when you reconnect

Reply:
oh github collapsed code and I thought it is compositionComplete ;-)